src/App.jsx

```diff
@@ -14,70 +16,52 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  const isSelectedGood = selectedGood ? `${selectedGood} is selected` : 'No goods selected';
 
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+  return (
+    <main className="section container">
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
+      <h1 className="title is-flex is-align-items-center">
+        {isSelectedGood}
+        {selectedGood && (<button onClick={() => setSelectedGood('')} data-cy="ClearButton" type="button" className="delete ml-3" />)}
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
+      </h1>
 
-        <tr data-cy="Good" className="has-background-success-light">
-          <td>
-            <button
-              data-cy="RemoveButton"
-              type="button"
-              className="button is-info"
-            >
-              -
-            </button>
-          </td>
+      <table className="table">
+        <tbody>
+          {goods.map((good) => {
+            const isGood = good === selectedGood;
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Jam
-          </td>
-        </tr>
+            return (
+              <tr className={isGood && 'has-background-success-light'} data-cy="Good">
+                <td>
+                  {isGood ? (
+                    <button
+                      data-cy="RemoveButton"
+                      type="button"
+                      className="button is-info"
+                      onClick={() => setSelectedGood('')}
+                    >
+                      -
+                    </button>
+                  ) : (
+                    <button onClick={() => setSelectedGood(good)} data-cy="AddButton" type="button" className="button">
+                      +
+                    </button>
+                  )}
```

Comment:
- You have inconsistent formatting for the attributes of the same button.  Also, when there are more than two attributes, it's better to have each attribute on a new line.
- You can improve the code for this button by using a ternary operator to assign the attribute.
-  For example: 
``` js
<button
      data-cy={isSelected
        ? 'RemoveButton'
        : 'AddButton'}